MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -30,6 +30,8 @@ public partial class ProbeResultNormalizer
         private const string ArtistReplaceValue = " | ";
 
         private readonly char[] _nameDelimiters = { '/', '|', ';', '\\' };
+        private readonly string[] _webmVideoCodecs = { "vp8", "vp9" };
```

Comment:
WebM unofficially supports AV1.
```suggestion
        private readonly string[] _webmVideoCodecs = { "av1", "vp8", "vp9" };
```